code/modules/mob/living/carbon/human/human_life.dm

```diff
@@ -12,6 +12,11 @@
 
 	voice = GetVoice()
 
+	// We clear this before we add extra in heartbeats and blood pressure
+	false_pain = 0
```

Comment:
I feel like, if you're making this variable as generic as `false_pain`, folks might be confused in the future when they try to add some kind of false pain and it doesn't stick around after a single life tick.

Maybe consider renaming it to something like `false_cardiac_pain`, or keeping track of multiple false pain variables? Iunno what the perfect solution would be